src/EntityFramework/EntityFramework.csproj

```diff
@@ -3,18 +3,26 @@
   <PropertyGroup>
     <RootNamespace>System.Data.Entity</RootNamespace>
     <GenerateDocumentationFile>True</GenerateDocumentationFile>
-    <TargetFrameworks>net45;net40</TargetFrameworks>
+    <TargetFrameworks>net45;net40;netstandard2.1</TargetFrameworks>
```

Comment:
Do you want to keep support for net40 ?

Reply:
Yes. Otherwise we’d have to call it EF7 😉

Reply:
@ErikEJ We were originally going to remove it, but then when the question was asked the response was (paraphrasing), "not unless it can be justified by showing nobody uses it." At that point it became more work to remove it than leave it in. And since it's also technically a breaking change, we're not going to change it.